### PR TITLE
WRN-4409: Migrate IncrementSlider and Slider unit tests to Testing Library

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated unit tests from Enzyme to Testing Library for IncrementSlider and Slider components.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-4409

### CODE REVIEW

File: IncrementSlider/tests/IncrementSlider-specs.js [hunk 1]
```diff
@@ -53,198 +42,155 @@ describe('IncrementSlider Specs', () => {
...
-		const expected = icon;
-		const actual = incrementSlider.find(`.${css.decrementButton} Icon`).prop('children');
+		const expected = '-';
+		const actual = screen.getAllByRole('button')[0];
 
-		expect(actual).toBe(expected);
+		expect(actual).toHaveTextContent(expected);
 	});
 
 	test(
```
Comment (alexandrumorariu) [inline IncrementSlider/tests/IncrementSlider-specs.js:103]: Write test(....) in a single line(same below)

File: Slider/tests/Slider-specs.js [hunk 2]
```diff
@@ -1,454 +1,389 @@
...
+		expect(slider).toHaveAttribute(expectedAttribute, expectedValue);
 	});
 
 	// these tests validate behavior relating to `value` defaulting to `min`
 	test('should not emit onChange when decrementing at the lower bound when value is unset', () => {
 		const handleChange = jest.fn();
-		const subject = mount(
-			<Slider min={0} max={10} onChange={handleChange} />
-		);
+		render(<Slider active min={0} max={10} onChange={handleChange} />);
```
Comment (alexandrumorariu) [inline Slider/tests/Slider-specs.js:199]: Order props in alphabetical order.

File: Slider/tests/Slider-specs.js [hunk 3]
```diff
@@ -1,454 +1,389 @@
...
-		expect(actual).toBe(expected);
+		expect(handleChange).not.toBeCalled();
 	});
 
 	test('should increment from the lower bound when value is unset', () => {
 		const handleChange = jest.fn();
-		const subject = mount(
-			<Slider min={0} max={10} onChange={handleChange} />
-		);
+		render(<Slider active min={0} max={10} onChange={handleChange} />);
```
Comment (alexandrumorariu) [inline Slider/tests/Slider-specs.js:210]: Order props in alphabetical order

File: Slider/tests/Slider-specs.js [hunk 4]
```diff
@@ -1,454 +1,389 @@
...
-		expect(actual).toBe(expected);
+		expect(handleSpotlight).toBeCalledTimes(expected);
 	});
 
 	test('should call onSpotlightLeft on vertical slider at any value', () => {
 		const handleSpotlight = jest.fn();
-		const subject = mount(
-			<Slider defaultValue={50} orientation="vertical" onSpotlightLeft={handleSpotlight} />
-		);
+		render(<Slider defaultValue={50} orientation="vertical" onSpotlightLeft={handleSpotlight} />);
```
Comment (alexandrumorariu) [inline Slider/tests/Slider-specs.js:237]: Order props in alphabetical order(same below)

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Pull request merged**